institutions-api/src/main/scala/hmda/institution/api/http/InstitutionQueryHttpApi.scala

```diff
@@ -93,7 +93,7 @@ trait InstitutionQueryHttpApi extends HmdaTimeDirectives with InstitutionEmailCo
     path("institutions") {
       timedGet { uri =>
         parameter('domain.as[String]) { domain =>
-          val f = findByEmail(domain, currentYear)
+          val f = findByEmailAnyYear(domain)
```

Comment:
If `currentYear` isn't being used, lets remove this variable along with removing it from K8 files so it is clearly visible that institutions-api no longer makes use of the `currentYear` (which comes from timed guards config map)